automation/check-patch.setup.sh

```diff
@@ -25,3 +25,6 @@ cp -rf $(pwd)/* ${TMP_PROJECT_PATH}
 
 echo 'Exporting temporary project path'
 export TMP_PROJECT_PATH
+
+echo 'Exporting git+date image tag'
+export IMAGE_TAG=$(git log -1 --pretty=%h)-$(date +%s)
```

Comment:
We don't need this in any other place but in publish.sh, please move it there. If we keep it in setup, we would probably forget about its existence and risk making this script into a pile of unused variables and functions.

Reply:
(The "setup" is just to setup the environment and shared dependencies of all the targets using this)

Reply:
Yeah, one thing I realized now is that it would furthermore make it impossible to override the tags from outside the publish script .

Reply:
I think this now covers all scenarios: allows the image tag to be overridden by the script caller, and by default, the script overrides it w/ commit-id+date.